packages/logging/src/log.js

```diff
@@ -520,8 +526,6 @@ Log.prototype.warning = function(entry, options, callback) {
  * @param {object[]} options.labels - Labels to set on the log.
  * @param {object} options.resource - A default monitored resource for entries
  *     where one isn't specified.
- * @param {boolean} options.removeCircular - Replace circular references in an
```

Comment:
This is still a useful option on the individual write. I think we should keep this, and if not specified, default to the value assigned on the Log.